SlackAPI.Tests/JoinDirectMessageChannel.cs

```diff
@@ -24,7 +24,7 @@ public void ShouldJoinDirectMessageChannel()
             JoinDirectMessageChannelResponse actual = null;
 
             string userName = this.fixture.Config.DirectMessageUser;
-            string user = client.Users.First(x => x.name.Equals(userName, StringComparison.InvariantCultureIgnoreCase)).id;
+            string user = client.Users.First(x => x.name.Equals(userName, StringComparison.OrdinalIgnoreCase)).id;
```

Comment:
Invariant is not supported in .NetCore. And according to this [issue](https://github.com/dotnet/corefx/issues/9801), InvariantCultureIgnoreCase usage is often to what we want